### PR TITLE
New konnector vente-privee.com

### PR DESCRIPTION
**Pull request by SomeAverageDev**
I also fixed some translations

### CODE REVIEW

**Comment by ZeHiro** [comment]
@SomeAverageDev 
Can you rebase your you branch over master, there are commit from you APRR konnector.

File: server/konnectors/venteprivee.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+function buildNotifContent(requiredFields, bills, data, next) {
+  if (bills.filtered.length > 0) {
+    const localizationKey = 'notification vente-privee';
+    const options = {
+      smart_count: bills.filtered.length,
+    };
+    bills.notifContent = localization.t(localizationKey, options);
+  }
+
+  next();
```
Comment (ZeHiro) [inline server/konnectors/venteprivee.js]: can you add `return next()`

File: server/konnectors/venteprivee.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    let orderId = null;
+    let orderDate = null;
+    let orderAmount = null;
+    let matchs = null;
+
+    const trId = $(this).attr('id');
+    log.debug('trId:', trId);
+
+    if (typeof (trId) === 'undefined') {
+      // next tr
```
Comment (ZeHiro) [inline server/konnectors/venteprivee.js]: Can you delete the comment?

**Comment by SomeAverageDev** [comment]
I made some updates, I might have done git mixup again...sorry sometimes I don't understand what I am really doing...

**Comment by frankrousseau** [comment]
No problem, git is a little bit messy.

**Pull request closed without merging**